tests/include/igl/cut_to_disk.cpp

```diff
@@ -130,3 +130,16 @@ TEST_CASE("cut_to_disk: cube", "[igl]")
 
   cut_to_disk_helper::assert_is_disk(V, F, cuts);
 }
+
+TEST_CASE("cut_to_disk: annulus", "[igl][PR984][bug]") {
```

Comment:
I would remove the tags `[PR984]` and `[bug]`. It's no longer a bug once it's fixed, and I would rather put the url to the PR/github issue in a comment above.

Reply:
Done, however, we will lose the ability of running just this one test like the following:
```
$ ./build/libigl_tests [pr984]
```

Reply:
You can run individual tests using their name `cut_to_disk: annulus`. If you want to keep the tag, maybe write it as `#984` (more github-like) rather than `PR984`. But that seems pretty minor to me.